### ISSUE TITLE
Use contract function via the Tasit SDK to buy land using the Decentraland marketplace contracts

### ISSUE THREAD

**Issue by pcowgill** [tasitlabs/tasit#62]

**Comment by pcowgill** [comment]
@marcelomorgado This issue would probably move the project forward a long way in terms of end-to-end completeness, because it would force us to [publish the latest version of the Tasit SDK](https://github.com/tasitlabs/TasitSDK/issues/145) and show how it could be used from within a demo app.

For that reason, I think it's a strong next choice for an issue to work on. But since I have the publishing permissions for the project on npm still, it probably would make sense to work on together towards the end of your work day.

**Comment by pcowgill** [comment]
﻿﻿﻿This is about to be merged, so that will unblock this PR https://github.com/tasitlabs/TasitSDK/issues/145

**Comment by pcowgill** [comment]
@marcelomorgado Think this should be the next major issue to work on?

**Comment by pcowgill** [comment]
If not, maybe just the "updating the demo app to use the latest SDK version" part could be split out into a separate issue that we tackle first

**Comment by marcelomorgado** [comment]
I'm thinking to finish the issue: https://github.com/tasitlabs/TasitSDK/issues/91#issuecomment-457782599
With test cases that we'll use on the app and then go into on this one. Make sense?

**Comment by pcowgill** [comment]
@marcelomorgado Sure, that sounds good!

**Comment by marcelomorgado** [comment]
Is this issue can be closed?

**Comment by pcowgill** [comment]
@marcelomorgado Yep! Closing it now. Thanks! Great work.
